### PR TITLE
feat(upgrade): add nix build configuration for upgrade-job image

### PR DESCRIPTION
**Pull request by niladrih**
This depends on #122 .

### CODE REVIEW

**Comment by sinhaashish** [comment]
bors try

**Review by sinhaashish — APPROVED** [review]

**Review by Abhinandan-Purkait — APPROVED** [review]

File: scripts/release.sh [hunk 1]
```diff
@@ -65,8 +65,8 @@ HASH=`get_hash`
 BRANCH=`git rev-parse --abbrev-ref HEAD`
 BRANCH=${BRANCH////-}
 IMAGES=
-DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome"
-IMAGES_THAT_REQUIRE_HELM_CHART=("operators.upgrade")
+DEFAULT_IMAGES="exporters.metrics.pool operators.upgrade obs.callhome upgrade.job"
+IMAGES_THAT_REQUIRE_HELM_CHART=("operators.upgrade" "upgrade.job")
```
Comment (tiagolobocastro) [inline scripts/release.sh:69]: these 2 upgrades are only here until we replace the old ugrade right?

Comment (sinhaashish) [reply scripts/release.sh:69]: correct

**Comment by sinhaashish** [comment]
bors merge

**Pull request merged**